api/health/service.go

```diff
@@ -22,6 +23,10 @@ type APIReply struct {
 	Healthy bool              `json:"healthy"`
 }
 
+type HealthArgs struct {
+	SubnetIDs []ids.ID `json:"subnetIDs"`
```

Comment:
Does it make sense for us to take in `ids.ID` here rather than `string`? Feels a bit odd for all of the code to be generic but for the API to be specific.

Reply:
Initially I was going for full generic param `Tags` []string. Then I thought it might be too much generic. However it's probably best to make them generic since we will use these for liveness/readiness checks as well.